opsimulator/opsimulator.go

```diff
@@ -510,7 +515,28 @@ func (opSim *OpSimulator) checkInteropInvariants(ctx context.Context, logs []typ
 	return nil
 }
 
-// Overridden such that the port field can appropiately be set
+// ValidateDependencySet checks if the source chain is allowed to send messages to the executing chain
+func (opSim *OpSimulator) ValidateDependencySet(sourceChainID uint64) error {
```

Comment:
similar comment here, it would be preferable to keep this function internal to the opsimulator since it is not needed by other packages, unless there is a good reason for it being public. Sorry for not catching this on first review.

Reply:
done!